hooks.ts

```diff
@@ -1,10 +1,12 @@
-const fs = require('fs');
-const path = require('path');
+import fs = require('fs');
```

Comment:
I've never seen this syntax before. Does this actually work?

Reply:
It seems?! Copied from the `os-launchpad` `hooks.ts` file

Reply:
Yep, does what I expected. Which makes it werid that it worked in the first place. I think JSON.parse must specifically check for buffer objects and handle appropriately, but that's not reflected in the types

Reply:
Hmmm. I really don't like it, but if it works then ok I guess.